toolchest_client/files/split.py

```diff
@@ -26,7 +26,7 @@ def open_new_output_file(
     if not os.path.exists(working_directory):
         os.mkdir(working_directory)
     current_output_file_path = f"{working_directory}/{filename_prefix}_{current_split_number}_{input_basename}"
-    return current_output_file_path, open(current_output_file_path, "w")
+    return current_output_file_path, open(current_output_file_path, "w", newline="\n")
```

Comment:
This is to make tests cooperate on Windows. `open()` on Windows will auto-write Windows-style `\r\n` instead of `\n` for line endings.